libs/components/lists/src/lib/modules/shared/filter-adapter/filter-adapter-data-change.ts

```diff
@@ -0,0 +1,10 @@
+import { SkyFilterAdapterData } from './filter-adapter-data';
+
+/**
+ * Represents a change to the filter adapter data including the source that triggered the change.
+ * @internal
+ */
+export interface SkyFilterAdapterDataChange {
+  data: SkyFilterAdapterData;
+  source: string;
```

Comment:
The methods have a parameter named `sourceId`. We can use that same name here, so the correlation is more obvious.